### PR TITLE
Security center policy change

### PR DESCRIPTION
**Pull request by prjelesi**
## Overview/Summary

update on policy and policy initiative based on 

https://dev.azure.com/CSUSolEng/Azure%20Landing%20Zones/_workitems/edit/25037/ 

Replace this with a brief description of what this Pull Request fixes, changes, etc.

This PR:

- Change display name & description to 'Deploy Microsoft Defender for Cloud Security Contacts' 
 - adds possibility to change 'minimalSeverity' property as input parameter for custom policy with name 'Deploy Microsoft Defender for Cloud Security Contacts'. It was previously hardcoded with value "high"
 - add minimalSeverity as input parameter for Deploy-MDFC-Config, Deploy-MDFC-Config (US Gov), Deploy-MDFC-Config (China) initiatives. 

### Breaking Changes
No breaking changes

## Testing Evidence

Test done manually

[Deploy To Azure](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fprjelesi%2FEnterprise-Scale%2FSecurity-Center-policy-change%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2Fprjelesi%2FEnterprise-Scale%2FSecurity-Center-policy-change%2FeslzArm%2Feslz-portal.json)

## As part of this Pull Request I have

- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [x] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [ ] Performed testing and provided evidence.
- [ ] Updated relevant and associated documentation.
- [ ] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

File: src/resources/Microsoft.Authorization/policyDefinitions/Deploy-ASC-SecurityContacts.json [hunk 1]
```diff
@@ -63,6 +76,10 @@
                 "field": "Microsoft.Security/securityContacts/email",
                 "contains": "[[parameters('emailSecurityContact')]"
               },
+              {
+                "field": "Microsoft.Security/securityContacts/alertNotifications.minimalSeverity",
+                "contains": "[parameters('minimalSeverity')]"
```
Comment (jtracey93) [inline src/resources/Microsoft.Authorization/policyDefinitions/Deploy-ASC-SecurityContacts.json:81]:
```suggestion
                "contains": "[[parameters('minimalSeverity')]"
```
This should be double escaped for ALZ requirements

**Review by jtracey93 — CHANGES_REQUESTED** [review]
Looks good apart from the one change requested.

**Comment by jtracey93** [comment]
@prjelesi can you pull in from main and resolve the conflict again please as @krowlandson has just merged #1132

**Comment by jtracey93** [comment]
@prjelesi can you also:

1. Update the Whats New wiki page in `docs/wiki` with these changes
2. Add a default value for the new parameter in the initiative as per #1132 did

Thanks

**Comment by prjelesi** [comment]
> @prjelesi can you also:
> 
> 1. Update the Whats New wiki page in `docs/wiki` with these changes
> 2. Add a default value for the new parameter in the initiative as per [Patch incorrect policy initiatives #1132](https://github.com/Azure/Enterprise-Scale/pull/1132) did
> 
> Thanks

#jtracey93 updated as per [Patch incorrect policy initiatives #1132](https://github.com/Azure/Enterprise-Scale/pull/1132) and tested.

policy01


i will proceed with wiki update now.

**Review by krowlandson — APPROVED** [review]
LGTM 👍🏻

**Review by jtracey93 — APPROVED** [review]

**Pull request merged**